universal-application-tool-0.0.1/app/services/program/ProgramDefinition.java

```diff
@@ -69,6 +71,30 @@ public static Builder builder() {
   /** The list of {@link ExportDefinition}s that make up the program. */
   public abstract ImmutableList<ExportDefinition> exportDefinitions();
 
+  /** When was this program created. Could be null for older programs. */
+  public abstract @Nullable Instant lastModifiedTime();
```

Comment:
We want to prefer Optional over null as it makes the need to check for "is missing" clear and obvious.

We can't easily do that in the models files, or in data we serialize, but the sooner in the stack that we can start using Optional the better; like here.